mu-plugins/blocks/language-suggest/src/block.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "wporg/language-suggest",
+	"version": "0.1.0",
+	"title": "Language Suggest",
+	"category": "widgets",
+	"icon": "admin-site-alt3",
+	"description": "A block for use across the whole wp.org network.",
+	"textdomain": "language-suggest",
+	"editorScript": "file:./index.js",
+	"viewScript": "file:./front.js",
+	"style": "file:./style.css",
+	"supports": {
+		"align": true,
```

Comment:
@ryelle Do you know why this isn't loading the full width alignment in `wporg-main-2022`? Based on [this documentation](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#align), it should work?

<img width="701" alt="Screen Shot 2022-07-29 at 11 38 12 AM" src="https://user-images.githubusercontent.com/1657336/181671476-6376addd-7d38-4ef2-b029-fe8e0e11155d.png">

Reply:
Huh, I got it to be full-width when testing on `main-test` with my sandbox:

<img width="384" alt="Screen Shot 2022-08-01 at 5 34 23 PM" src="https://user-images.githubusercontent.com/541093/182250364-da140603-4263-4190-94ae-3d866df43747.png">

Maybe you've got it in a container that doesn't support wide/full alignment?